mlir-sycl/test/Dialect/SYCL/aux-ops.mlir

```diff
@@ -29,3 +29,59 @@ func.func @test_folder(%arg0 : f16, %arg1 : !sycl_half) -> (!sycl_half, f16) {
   // FOLD: return %[[wrap]], %[[unwrap]] : !sycl_half, f16
   return %2, %5 : !sycl_half, f16
 }
+
+!sycl_vec_i32_4_ = !sycl.vec<[i32, 4], (vector<4xi32>)>
+
+// CHECK-LABEL: test_vector_of_native
```

Comment:
`CHECK-LABEL` doesn't match the name of the function, should probably remove the second `test_` from the function name (also below).

Reply:
Oops, delicious copy-n-pasta, thanks!